examples/download_and_train.py

```diff
@@ -10,7 +11,7 @@
 # Download the 3D multi-channel tiffs via Quilt/T4
 ###################################################
 
-gpu_id = 0
+gpu_id = 7
```

Comment:
Unless the actual fnet application code has drastically changed, I believe this should always be zero for most users.

```suggestion
gpu_id = 0
```